SAPHana/ra/SAPHanaController

```diff
@@ -2364,8 +2379,19 @@ function saphana_monitor_primary()
             fi
             ;;
         2 | 3 | 4 ) # WARN, INFO or OK
+            ATTR=(${ATTR_NAME_HANA_LSS[@]}); ATTR[0]="${ATTR[0]}_$sr_name"
+            my_lss=$(get_hana_attribute "X" "${ATTR[@]}")
+            ATTR=(${ATTR_NAME_HANA_SRR[@]}); ATTR[0]="${ATTR[0]}_$sr_name"
+            my_srr=$(get_hana_attribute "X" "${ATTR[@]}")
+            my_role=$(get_hana_attribute ${NODENAME} "${ATTR_NAME_HANA_ROLES[@]}")
+            # saphana_monitor_primary - get SR attribute of "the" remote site (TODO: PRIO3: allow multi targets in the
+            my_sync=$(get_SRHOOK "$remSR_name")
+            my_hana_role="${my_lss}:${my_srr}:${my_role}"
             if ocf_is_probe; then
                 rc=$OCF_SUCCESS
+                my_role=$(get_hana_attribute ${NODENAME} ${ATTR_NAME_HANA_ROLES[@]})
+                super_ocf_log info "SCORE: saphana_monitor_primary: (1) scoring_crm_master($my_hana_role,$my_sync)"
+                scoring_crm_master "$my_hana_role" "$my_sync"
```

Comment:
Willst Du wirklich '$my_hana_role' (aus Zeile 2389)  für das scoring nehmen? Warum wird dann aber in Zeile 2392 nochmal my_role gesetzt, wenn es danach nicht mehr genutzt wird?

Reply:
Neuer commit sollte das jetzt gefixed haben. Das Attribute muss nicht (nochmal) gelesen werden. Zum Scoren soll (hier) my_hana_role verwendet werden. Die Variable wird direct vor der Zeile 2390 gesetzt.